config/prod/phil-cough-data.yaml

```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: phil-cough-data
```

Comment:
ohh wait, i think you said this bucket will likely live longer than you?  do you really want to have your name on the resources then?  AWS will append this `stack_name` to all the resources it creates.

Reply:
Hahaha please just merge it 😂 This will be my legacy.